From 2ed391b4a849b4daf7d38d7b1341accc1eceedfa Mon Sep 17 00:00:00 2001
From: Pavel Raiskup <praiskup@redhat.com>
Date: Sun, 02 Nov 2014 10:53:20 +0000
Subject: libtoolize: do not remove gnulib files with --force.

* libtoolize.in (func_require_seen_libtool): Do not remove
snippet/* files which are from Gnulib.

Signed-off-by: Gary V. Vaughan <gary@gnu.org>
---
diff --git a/libtoolize.in b/libtoolize.in
index d819470..9fda071 100644
--- a/libtoolize.in
+++ b/libtoolize.in
@@ -1896,8 +1896,8 @@ func_require_seen_libtool ()
   # ensure a clean upgrade.
   # Do not remove config.guess, config.sub or install-sh, we don't
   # install them without --install, and the project may not be using
-  # Automake.
-  all_pkgaux_files="compile depcomp missing ltmain.sh snippet/_Noreturn.h snippet/arg-nonnull.h snippet/c++defs.h snippet/warn-on-use.h"
+  # Automake.  Similarly, do not remove Gnulib files.
+  all_pkgaux_files="compile depcomp missing ltmain.sh"
   all_pkgmacro_files="argz.m4 libtool.m4 ltdl.m4 ltoptions.m4 ltsugar.m4 ltversion.in ltversion.m4 lt~obsolete.m4"
   all_pkgltdl_files="COPYING.LIB Makefile Makefile.in Makefile.inc Makefile.am README acinclude.m4 aclocal.m4 argz_.h argz.c config.h.in config-h.in configure configure.ac configure.in libltdl/lt__alloc.h libltdl/lt__dirent.h libltdl/lt__glibc.h libltdl/lt__private.h libltdl/lt__strl.h libltdl/lt_dlloader.h libltdl/lt_error.h libltdl/lt_system.h libltdl/slist.h loaders/dld_link.c loaders/dlopen.c loaders/dyld.c loaders/load_add_on.c loaders/loadlibrary.c loaders/preopen.c loaders/shl_load.c lt__alloc.c lt__dirent.c lt__strl.c lt_dlloader.c lt_error.c ltdl.c ltdl.h ltdl.mk slist.c"
 
--
cgit v0.9.0.2
